Client/qtTeamTalk/preferencesdlg.cpp

```diff
@@ -771,6 +771,39 @@ void PreferencesDlg::slotSaveChanges()
             if(lang != ttSettings->value(SETTINGS_DISPLAY_LANGUAGE).toString())
             {
                 switchLanguage(lang);
+                QMessageBox answer;

```

Comment:
This dialog box should only be displayed if there are already active TTS events or Status messages. Otherwise people will not understand why it is shown.